firmware/tunerstudio/rusefi.input

```diff
@@ -4028,11 +4028,11 @@ dialog = tcuControls, "Transmission Settings"
 
 	dialog = etbAutotune, "PID Autotune"
 		field = "First step: calibrate TPS and hit 'Burn'"
-		commandButton = "Auto Calibrate ETB 1", cmb_etb_auto_calibrate, {etb1configured == 1 && tps1_1AdcChannel != @@ADC_CHANNEL_NONE@@}
-		commandButton = "Auto Calibrate ETB 2", cmb_etb_auto_calibrate_2, {etb2configured == 1 && tps2_1AdcChannel != @@ADC_CHANNEL_NONE@@}
+		commandButton = "Auto Calibrate ETB 1", cmd_etb_auto_calibrate, {etb1configured == 1 && tps1_1AdcChannel != @@ADC_CHANNEL_NONE@@ && calibrationMode == @@TsCalMode_None@@}
+		commandButton = "Auto Calibrate ETB 2", cmd_etb_auto_calibrate_2, {etb2configured == 1 && tps2_1AdcChannel != @@ADC_CHANNEL_NONE@@ && calibrationMode == @@TsCalMode_None@@}
 		field = "Second step"
-		commandButton = "Start ETB PID Autotune", cmd_etb_autotune,       { etb1configured == 1 || etb2configured == 1 }
-		commandButton = "Stop ETB PID Autotune",  cmd_etb_autotune_stop,  { etb1configured == 1 || etb2configured == 1 }
+		commandButton = "Start ETB PID Autotune", cmd_etb_autotune,       {(etb1configured == 1 || etb2configured == 1) && calibrationMode = @@TsCalMode_None@@}
+		commandButton = "Stop ETB PID Autotune",  cmd_etb_autotune_stop,  {(etb1configured == 1 || etb2configured == 1) && calibrationMode >= @@TsCalMode_EtbKp@@ && calibrationMode <= @@TsCalMode_EtbKd@@}
```

Comment:
The condition in parenthesis is redundant now, as 3-5 imply ETB calib underway.  It also may cause a problem if somehow the calibration is enabled and they change: cannot be disabled (though that's always been the case): shall I remove it?